collectibles/collectibles.py

```diff
@@ -0,0 +1,223 @@
+# Copyright (c) 2021 - Jojo#7791
+# Licensed under MIT
+
+import logging
+from datetime import datetime
+from typing import Literal, Optional, Union
+
+import discord
+from jojo_utils import Menu, positive_int
+from redbot.core import Config, bank, commands
+from redbot.core.bot import Red
+from redbot.core.utils.chat_formatting import pagify
+
+from .utils import Page
+
+log = logging.getLogger("red.JojoCogs.collectibles")
+log.setLevel(logging.DEBUG)
+_config_structure = {
+    "user": {
+        "collectibles": {},
+        "total_count": 0,
+    },
+    "guild": {
+        "collectibles": {},
+    },
+    "global": {"collectibles": {"Trophy": ["🏆", 300]}},
+}
+
+
+class Collectibles(commands.Cog):
+    """Collect trinkets and items!"""
+
+    __author__ = ["Jojo#7791"]
+    __version__ = "1.0.0"
+
+    def __init__(self, bot: Red):
+        self.bot = bot
+        self.config = Config.get_conf(self, 544974305445019651, True)
+        for key, value in _config_structure.items():
+            getattr(self.config, f"register_{key}")(
+                **value
+            )  # am too lazy to type it all out :)
```

Comment:
I mean this works but I dunno if Red QA will like it...

Reply:
That's fair lol